src/System.Linq.Expressions/tests/ListInit/ListInitExpressionTests.cs

```diff
@@ -214,5 +214,15 @@ public void UpdateDifferentInitializersReturnsDifferent()
             var init = Expression.ListInit(Expression.New(typeof(List<int>)), inits);
             Assert.NotSame(init, init.Update(Expression.New(typeof(List<int>)), inits));
         }
+
+        [Fact]
+        public static void ToStringTest()
+        {
+            var e1 = Expression.ListInit(Expression.New(typeof(List<int>)), Expression.Parameter(typeof(int), "x"));
+            Assert.Equal("new List`1() {Void Add(Int32)(x)}", e1.ToString());
```

Comment:
List`1   is a typical "reflection style" printout. I think it is actually the name of the type. Compilers by convention add `< arity > to generic definitions.
However, I wonder why it is not "List`1< Int32 >"